README.md

```diff
@@ -161,6 +161,14 @@ Use this method to set your feed's URL.
 const cal = ical().url('https://example.com/calendar.ical');
 ```
 
+#### calscale([_String_ scale])
+
+Use this method to set your feed's CALSCALE.
```

Comment:
Please add a small sentence that notes that there is no default value for this method from ical-generator, but the ical-standard is then takes the gregorian calendar.